clouddeploy/sso-dashboard-prod.template.yaml

```diff
@@ -23,8 +23,8 @@ spec:
   template:
     metadata:
       annotations:
-        autoscaling.knative.dev/minScale: '1'
-        autoscaling.knative.dev/maxScale: '3'
+        autoscaling.knative.dev/minScale: '0'
```

Comment:
Relevant to the comment about the dev config, having minScale at zero leads to cold starts and scaling delays. Might be worth the cost savings, dunno